api/app/src/routes/internal.ts

```diff
@@ -176,4 +179,42 @@ router.post(
   })
 );
 
+const statusSchema = z.enum(["success", "failed"]);
+
+router.post(
+  "/doc-conversion-status",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patientId = getUUIDFrom("query", req, "patientId").orFail();
+    const status = getFrom("query").orFail("status", req);
+    const convertStatus = statusSchema.parse(status);
+
+    const patient = await getPatientOrFail({ id: patientId, cxId });
+
+    const convertSuccess = patient.data.documentQueryProgress?.convertSuccess
+      ? patient.data.documentQueryProgress.convertSuccess + 1
+      : 1;
+
+    const convertError = patient.data.documentQueryProgress?.convertError
+      ? patient.data.documentQueryProgress.convertError + 1
+      : 1;
```

Comment:
I think we should move this to within the DB tx, otherwise we can a racing condition where two requests increase at the same time, and the result will be lower than what we'd expect. Likely have this on `updateDocQuery()`.